doc/en/components/grids/_shared/state-persistence.md

```diff
@@ -93,39 +108,141 @@ const gridState: IGridState = state.getState(false);
 // get the sorting and filtering expressions
 const sortingFilteringStates: IGridState = state.getState(false, ['sorting', 'filtering']);
 ```
+<!-- end: Angular -->
+
+<!-- WebComponents -->
+```html
+<{ComponentSelector} id="grid">
+    <igc-grid-state id="gridState"></igc-grid-state>
+</{ComponentSelector}>
+```
+
+```typescript
+var gridState = document.getElementById('gridState') as IgcGridStateComponent;
+
+// get an `IgcState` object, containing all features original state objects, as returned by the grid public API
+const state: IgcState = gridState.extractState();
```

Comment:
Some things will probably need to be renamed like:

`IgcState` -> `IgcStateInfo`
`extractState`/`extractStateAsString` -> `getState`/`getStateAsString`